### DISCUSSION TITLE
How to connect/cable device from one rack to another device on another rack

### DISCUSSION THREAD

**Discussion by iftam** [netbox-community/netbox#16168] [Help Wanted!]
I don't know if it's even possible, is there a way I can connect/cable two devices that is not in the same rack? For example, I want to connect g0/1 interface from switch A in rack 1 to switch B, interface g0/3 in rack 2. So far I can find a way I can directly connect these ports without creating a patch panel or some kind of termination point in each rack. Am I missing something?

**Comment by candlerb** [comment]
> I don't know if it's even possible, is there a way I can connect/cable two devices that is not in the same rack?

Yes. Simply create a cable with the relevant two endpoints. Nothing constraints the endpoints to be in the same rack, or even  the same site.

What version of Netbox are you using? Maybe it's a misunderstanding about how the GUI works. Older versions of Netbox would preselect the rack for the other end of the cable to be the same rack as the first end, but you can simply change it to a different rack.

3.7.x and 4.0.x give you a list of *all* devices to select from, plus a separate button to filter this down (which IMO is harder to use if you have a large number of devices, especially if you re-use the same device name across multiple sites). You can try out 4.0.x at https://demo.netbox.dev/
